From 110a96d728d54ff31d0d5ff1684427b45e91b832 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 28 Jul 2023 11:26:49 +0100
Subject: [PATCH] curl: Make times seconds field slightly wider

Updates: commit 68dddbeb584fb9385915846d259563f74338ffe8
(cherry picked from commit 1622db8c9f6c6e42b679d1d81df019944178b373)
---
 plugins/curl/nbdkit-curl-plugin.pod | 14 +++++++-------
 plugins/curl/times.c                |  2 +-
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/plugins/curl/nbdkit-curl-plugin.pod b/plugins/curl/nbdkit-curl-plugin.pod
index 0fd688ed..0774adad 100644
--- a/plugins/curl/nbdkit-curl-plugin.pod
+++ b/plugins/curl/nbdkit-curl-plugin.pod
@@ -551,13 +551,13 @@ information is printed in the debug output before nbdkit exits.  The
 output will look like:
 
  nbdkit: debug: times (-D curl.times=1):
- nbdkit: debug: name resolution               :   0.128442 s
- nbdkit: debug: connection                    :   4.945213 s
- nbdkit: debug: SSL negotiation               :   4.291362 s
- nbdkit: debug: pretransfer                   :   0.104137 s
- nbdkit: debug: first byte received           :  56.115269 s
- nbdkit: debug: data transfer                 : 222.633831 s
- nbdkit: debug: redirection time              :   0.000000 s
+ nbdkit: debug: name resolution               :    0.128442 s
+ nbdkit: debug: connection                    :    4.945213 s
+ nbdkit: debug: SSL negotiation               :    4.291362 s
+ nbdkit: debug: pretransfer                   :    0.104137 s
+ nbdkit: debug: first byte received           :   56.115269 s
+ nbdkit: debug: data transfer                 :  222.633831 s
+ nbdkit: debug: redirection time              :    0.000000 s
 
 The cumulative time taken to perform each step is shown (summed across
 all HTTP connections).  The redirection time is the total time taken
diff --git a/plugins/curl/times.c b/plugins/curl/times.c
index 8cc4cf27..e752a0a9 100644
--- a/plugins/curl/times.c
+++ b/plugins/curl/times.c
@@ -123,7 +123,7 @@ display_times (void)
       v = t;
     prev_t = t;
 
-    nbdkit_debug ("%-30s: %3" PRIi64 ".%06" PRIi64 " s",
+    nbdkit_debug ("%-30s: %4" PRIi64 ".%06" PRIi64 " s",
                   times[i].name,
                   v / 1000000, v % 1000000);
   }
-- 
2.39.3

